src/sentry/incidents/utils/metric_issue_poc.py

```diff
@@ -61,19 +67,35 @@ def construct_title(alert_rule: AlertRule) -> str:
     else:
         higher_or_lower = "less than" if alert_rule.comparison_delta else "below"
 
+    label = INCIDENT_STATUS[IncidentStatus(status)].lower()
+
     # Format the time window for the threshold
     time_window = alert_rule.snuba_query.time_window // 60
-    title = f"{agg_text} in the last {format_duration_idiomatic(time_window)} {higher_or_lower}"
+    title = f"{label.capitalize()}: {aggregate}"
 
     # If the alert rule has a comparison delta, format the comparison string
     if alert_rule.comparison_delta:
         comparison_delta_minutes = alert_rule.comparison_delta // 60
         comparison_string = TEXT_COMPARISON_DELTA.get(
             comparison_delta_minutes, f"same time {comparison_delta_minutes} minutes ago"
         )
-        return _(f"{title} {comparison_string}")
+        return _(
+            f"{title} in the last {format_duration_idiomatic(time_window)} {higher_or_lower} {comparison_string}"
+        )
+
+    trigger = AlertRuleTrigger.objects.filter(id=alert_rule.id, label=label.lower()).first()
+    if not trigger:
+        return _(
+            f"{title} in the last {format_duration_idiomatic(time_window)} {higher_or_lower} threshold"
+        )
+
+    threshold = trigger.alert_threshold
+    if threshold % 1 == 0:
+        threshold = int(threshold)
```

Comment:
why can't we use `format_duration_idiomatic` here as well?

Reply:
`format_duration_idiomatic` is to format the minutes into a string, but threshold is a number that could refer to number of errors, users affected, etc.